packages/odyssey-react-mui/src/Tag.tsx

```diff
@@ -0,0 +1,41 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Chip } from ".";
+import { memo, useContext } from "react";
+import { TagListContext } from "./TagListContext";
+
+export type TagProps = {
+  isDisabled?: boolean;
+  label: string;
+  onClick?: (event: React.MouseEvent<HTMLDivElement, MouseEvent>) => void;
+  onDelete?: (event: React.MouseEvent<HTMLDivElement, MouseEvent>) => void;
+};
+
+const Tag = ({ isDisabled, label, onClick, onDelete }: TagProps) => {
```

Comment:
What'd you think about `onRemove` vs `onDelete`?

Delete makes me think the tag's gone completely. Here, it's only removed as a potential value.

Reply:
> Delete makes me think the tag's gone completely. Here, it's only removed as a potential value.

This is true for `Autocomplete` but may not be accurate for other uses of `Tag` -- for instance, if you are editing an Application in the OIN and remove the `O365` tag.

No problem with the rename, just want to flag the dual use case.